### PR TITLE
feat: add online speaker images and names

### LINKED ISSUE #408: Add Online Speaker Images
### Why do we need this improvement?

Start adding online speakers images on the conference website.These are the speakers that will speak during the online conference

### How could it be implemented/designed?

This is the current list of the speakers and get their images from linkedIN:

1. Hari Krishnan - https://www.linkedin.com/in/harikrishnan83/
2. Yuan Yuan - https://www.linkedin.com/in/yuanyuanvassar/
3. Ivan Garcia Sainz-Aja - https://www.linkedin.com/in/ivangarciasainzaja/
4. Atinuke Oluwabamikemi Kayode - https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
5. Aishat Muibudeen - https://www.linkedin.com/in/aishatmuibudeen
6. Thulisile Sibanda - https://www.linkedin.com/in/v-thulisile-sibanda/
7. Ankit Dash - https://github.com/helios2003
8. Giri Venkatesan - https://www.linkedin.com/in/girivenkatesan/

### PR DESCRIPTION
**Pull request by techmannih**
Close #408 
Issue #408 
@thulieblack please review and merge it.
Screenshot 2024-10-02 222842

### CODE REVIEW

**Comment by ashmit-coder** [comment]
hey @techmannih could you into why some of the checks are failing?

**Comment by ashmit-coder** [comment]
Hey @thulieblack shall we make the images available offline, one pr is dealing with that issue. If we are adding on new images I think it should follow the same convention.

**Comment by thulieblack** [comment]
The pr is not complete, please add the descriptions for each.

**Comment by thulieblack** [comment]
Yes @ashmit-coder it should, please give some details on how too

**Comment by ashmit-coder** [comment]
hey @thulieblack @techmannih 
https://github.com/asyncapi/conference-website/pull/404

lets wait for this issue to get resolved at first IMO. 
If this pr is urgent then lets merge it (after it passes the checks) and later we will raise a new issue to follow the local images convention.

**Comment by thulieblack** [comment]
Makes sense, the PR isn't ready yet some of the speakers info is missing that's why checks aren't passing

**Comment by techmannih** [comment]
@ashmit-coder  @thulieblack  there is no description for rest of speakers, so how will I add

**Comment by ashmit-coder** [comment]
Lets see I am not quite sure why the checks are failing as well.

**Comment by ashmit-coder** [comment]
> @ashmit-coder @thulieblack there is no description for rest of speakers, so how will I add

I guess @thulieblack means adding the speakers from the last years session ?

**Comment by techmannih** [comment]
> hey @techmannih could you into why some of the checks are failing?



> Lets see I am not quite sure why the checks are failing as well.

these are passed now

**Comment by ashmit-coder** [comment]
Hey @techmannih , the titles for
1. Atinuke Oluwabamikemi Kayode
2. Aishat Muibudeen
3. Ankit Dash

are missing.

File: config/speakers.json [hunk 1]
```diff
@@ -170,7 +170,46 @@
 	{
 		"city": "Online",
 		"location": "Online Edition",
-		"lists": [],
+		"lists": [			{
+			"name": "Hari Krishnan",
+			"title": "Founder and CEO at Polarizer Technologies",
```
Comment (thulieblack) [inline config/speakers.json:175]:
```suggestion
			"title": "Co-founder, CTO at Specmatic",
```

File: config/speakers.json [hunk 2]
```diff
@@ -170,7 +170,46 @@
...
+			"img": "https://media.licdn.com/dms/image/v2/D5603AQHDfk4HvRiZFQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1711019429082?e=1733356800&v=beta&t=Tr5uWSEbmXU-tTQkGCE7Q2aJeBW1jZzFJwY-d6bOxT8"
+		},
+		{
+			"name": "Ankit Dash",
+			"title": "",
+			"img": "https://media.licdn.com/dms/image/v2/D5603AQEKgTNAVwxuHg/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1687942191582?e=1733356800&v=beta&t=GVsvgRB0kV_dkHkkEdyZ3h9_KqxBZ6sgeyio4iaurhQ"
+		},
+		{
+			"name": "Giri Venkatesan",
+			"title": "Principal Developer Advocate, Office of the CTO at Solace",
```
Comment (thulieblack) [inline config/speakers.json:210]:
```suggestion
			"title": "Principal Developer Advocate and Architect at Solace at Solace",
```

File: config/speakers.json [hunk 3]
```diff
@@ -170,7 +170,46 @@
...
+			"img": "https://media.licdn.com/dms/image/v2/C5603AQHQRFnT-j9V0w/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1657787935712?e=1733356800&v=beta&t=dd3I1-lgNTW1BjOpLBljK8TYaAYcTtng3XvmsS2Dpuw"
+		},
+		{
+			"name": "Aishat Muibudeen",
+			"title": "",
+			"img": "https://media.licdn.com/dms/image/v2/D4E03AQEX1YDxXNXWYw/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1711890580471?e=1733356800&v=beta&t=-_VSVmb9JS1HA6BP4k2BEDpw6Eg0bF-VddG31j2PE1Q"
+		},		
+		{
+			"name": "Thulisile Sibanda",
+			"title": "",
```
Comment (thulieblack) [inline config/speakers.json:200]:
```suggestion
			"title": "Community Manager at AsyncAPI Initiative",
```

File: config/speakers.json [hunk 4]
```diff
@@ -170,7 +170,46 @@
...
+			"img": "https://avatars.githubusercontent.com/u/16447041?v=4"
+		},
+		{
+			"name": "Ivan Garcia Sainz-Aja",
+			"title": "Java Architect. Speaker. Building ZenWave 360°",
+			"img": "https://media.licdn.com/dms/image/v2/C4E03AQHwWMQkufyIxQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1517669088843?e=1733356800&v=beta&t=pnX1RAueiwPNPuyLkgAHW4KexvNB0bSKv6vDTWIgDsA"
+		},
+		{
+			"name": "Atinuke Oluwabamikemi Kayode",
+			"title": "",
```
Comment (thulieblack) [inline config/speakers.json:190]:
```suggestion
			"title": "Community Marketing Specialist at AsyncAPI Initiative",
```

File: config/speakers.json [hunk 5]
```diff
@@ -170,7 +170,46 @@
...
+			"img": "https://media.licdn.com/dms/image/v2/D4E03AQEX1YDxXNXWYw/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1711890580471?e=1733356800&v=beta&t=-_VSVmb9JS1HA6BP4k2BEDpw6Eg0bF-VddG31j2PE1Q"
+		},		
+		{
+			"name": "Thulisile Sibanda",
+			"title": "",
+			"img": "https://media.licdn.com/dms/image/v2/D5603AQHDfk4HvRiZFQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1711019429082?e=1733356800&v=beta&t=Tr5uWSEbmXU-tTQkGCE7Q2aJeBW1jZzFJwY-d6bOxT8"
+		},
+		{
+			"name": "Ankit Dash",
+			"title": "",
```
Comment (thulieblack) [inline config/speakers.json:205]:
```suggestion
			"title": "Full Stack Developer",
```

File: config/speakers.json [hunk 6]
```diff
@@ -170,7 +170,46 @@
...
+			"img": "https://media.licdn.com/dms/image/v2/C4E03AQHwWMQkufyIxQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1517669088843?e=1733356800&v=beta&t=pnX1RAueiwPNPuyLkgAHW4KexvNB0bSKv6vDTWIgDsA"
+		},
+		{
+			"name": "Atinuke Oluwabamikemi Kayode",
+			"title": "",
+			"img": "https://media.licdn.com/dms/image/v2/C5603AQHQRFnT-j9V0w/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1657787935712?e=1733356800&v=beta&t=dd3I1-lgNTW1BjOpLBljK8TYaAYcTtng3XvmsS2Dpuw"
+		},
+		{
+			"name": "Aishat Muibudeen",
+			"title": "",
```
Comment (thulieblack) [inline config/speakers.json:195]:
```suggestion
			"title": "Product Designer and UX Researcher",
```

File: config/speakers.json [hunk 7]
```diff
@@ -170,7 +170,46 @@
...
+			"img": "https://media.licdn.com/dms/image/v2/C5603AQECQuUSsOONpQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1652006613665?e=1733356800&v=beta&t=KmRc3uH5PZOBemSNDS08prfAOLxJGli21TA35jt5oog"
+		},
+		{
+			"name": "Yuan Yuan",
+			"title": "MSE in DS Student at the University of Pennsylvania",
+			"img": "https://avatars.githubusercontent.com/u/16447041?v=4"
+		},
+		{
+			"name": "Ivan Garcia Sainz-Aja",
+			"title": "Java Architect. Speaker. Building ZenWave 360°",
```
Comment (thulieblack) [inline config/speakers.json:185]:
```suggestion
			"title": "Java Architect",
```

File: config/speakers.json [hunk 8]
```diff
@@ -170,7 +170,46 @@
...
 		"location": "Online Edition",
-		"lists": [],
+		"lists": [			{
+			"name": "Hari Krishnan",
+			"title": "Founder and CEO at Polarizer Technologies",
+			"img": "https://media.licdn.com/dms/image/v2/C5603AQECQuUSsOONpQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1652006613665?e=1733356800&v=beta&t=KmRc3uH5PZOBemSNDS08prfAOLxJGli21TA35jt5oog"
+		},
+		{
+			"name": "Yuan Yuan",
+			"title": "MSE in DS Student at the University of Pennsylvania",
```
Comment (thulieblack) [inline config/speakers.json:180]:
```suggestion
			"title": "Software Engineer",
```

**Review by thulieblack — CHANGES_REQUESTED** [review]
@techmannih please make the changes

**Comment by techmannih** [comment]
@ashmit-coder @thulieblack  please check now

**Comment by thulieblack** [comment]
@techmannih, the images are not showing


https://github.com/user-attachments/assets/23614a2c-7d61-4b15-a9f8-b5c859e396ac

I suggest you download them, create a folder locally under `public/img/online-conf-images/` and reference them using the following path `/img/online-images/.....jpeg`

**Comment by techmannih** [comment]
@ashmit-coder @thulieblack

**Comment by thulieblack** [comment]
Thanks, @techmannih; I'll wait for @ashmit-coder's review first.

Please be patient, reviews take time

**Comment by techmannih** [comment]
@thulieblack sure, thank you!

**Comment by ashmit-coder** [comment]
Hey @techmannih could you try to use `.webp` format for the images instead of `jpeg` . We are looking forward to reduce load times of the conference website as well and this will be a great start to it!
If you are unable to do it, just use jpeg. 
Other than that the changes LGTM @thulieblack ! (Just verify the data from your end)

File: config/speakers.json (hunk 1 above)
Comment (ashmit-coder) [reply config/speakers.json]: just resolve these conseversations as well @techmannih .

**Comment by techmannih** [comment]
@techmannih @thulieblack please check now

**Comment by ashmit-coder** [comment]
LGTM!

**Comment by ashmit-coder** [comment]
Hey @thulieblack ig the PR is ready to be merged!

**Review by thulieblack — APPROVED** [review]
Thanks @techmannih

**Comment by techmannih** [comment]
@thulieblack please add Hacktoberfest-accepted label

**Pull request merged**